spec/fx/definition_spec.rb

```diff
@@ -2,16 +2,17 @@
 
 describe Fx::Definition do
   describe "#to_sql" do
+    let(:sql_definition) { <<-EOS }
```

Comment:
I avoid using lets in this project as per https://thoughtbot.com/blog/lets-not.

Can you inline the lets into their corresponding tests?

Reply:
Sure!